tsconfig.node.tsbuildinfo

```diff
@@ -0,0 +1 @@
+{"root":["./vite.config.ts"],"version":"5.6.2"}
```

Comment:
We don't need these too...

it can always be generated locally for test when we need it

Reply:
https://vite-pwa-org.netlify.app/ 
I've use this plugin